common/v2/features/Dashboard/components/AccountList.tsx

```diff
@@ -18,45 +18,72 @@ const DeleteButton = styled(Button)`
   margin-left: 1em;
 `;
 
-export default function AccountList({ accounts, deleteAccount, className = '' }: Props) {
-  const accountTable = {
-    head: ['Favorite', 'Address', 'Network', 'Value', 'Delete'],
-    body: accounts.map(account => {
-      return [
-        <Icon key={0} icon="star" />,
-        <Address
-          key={1}
-          title={`${account.label} - (${account.accountType})`}
-          address={account.address}
-          truncate={truncate}
-        />,
-        <Network key={3} color="#a682ff">
-          {account.network}
-        </Network>,
-        <Typography key={4}>{account.value}</Typography>,
-        <DeleteButton key={5} onClick={() => deleteAccount(account.uuid)} icon="exit" />
-      ];
-    }),
-    config: {
-      primaryColumn: 'Address',
-      sortableColumn: 'Address',
-      sortFunction: (a: any, b: any) => {
-        const aLabel = a.props.label;
-        const bLabel = b.props.label;
-        return aLabel === bLabel ? true : aLabel.localeCompare(bLabel);
-      },
-      hiddenHeadings: ['Favorite', 'Delete'],
-      iconColumns: ['Favorite', 'Delete']
-    }
+export default class AccountList extends Component<Props> {
+  public handleAccountDelete = (uuid: string) => () => {
+    const { deleteAccount } = this.props;
+    deleteAccount(uuid);
   };
-  return (
-    <DashboardPanel
-      heading="Your Accounts"
-      action="Add Account"
-      actionLink="/dashboard/add-account"
-      className={`AccountList ${className}`}
-    >
-      <CollapsibleTable breakpoint={450} {...accountTable} />
-    </DashboardPanel>
-  );
+
+  public renderRedirect() {
+    if (
+      this.props.accounts === undefined ||
+      this.props.accounts === null ||
+      this.props.accounts.length === 0
+    ) {
+      return <Redirect to="/no-accounts" />;
+    }
+  }
+
+  public BuildAccountList() {
+    const { accounts } = this.props;
+    return {
+      head: ['Favorite', 'Address', 'Network', 'Value', 'Delete'],
+      body: accounts.map(account => {
+        return [
+          <Icon key={`account-icon${account.uuid}`} icon="star" />,
+          <Address
+            key={`account-key-${account.value}`}
+            title={`${account.label}- (${account.accountType})`}
+            address={account.address}
+            truncate={truncate}
+          />,
+          <Network key={`account-network-${account.network}`} color="#a682ff">
+            {account.network}
+          </Network>,
+          <Typography key={`account-value-${account.value}`}>{account.value}</Typography>,
+          <DeleteButton
+            key={`account-id-${account.uuid}`}
+            onClick={this.handleAccountDelete(account.uuid)}
+            icon="exit"
+          />
+        ];
```

Comment:
I don't think you need the key prop here because the Table component maps over children and adds them automatically (but our linter doesn't know this so it gets confused)